dbt_sugar/core/connectors/snowflake_connector.py

```diff
@@ -30,15 +30,28 @@ def __init__(
             connection_params (Dict[str, str]): Dict containing database connection
                 parameters and credentials.
         """
-        self.connection_url = URL(
-            drivername="postgresql+psycopg2",
-            user=connection_params.get("user", str()),
-            password=connection_params.get("password", str()),
-            database=connection_params.get("database", str()),
-            account=connection_params.get("account", str()),
-            warehouse=connection_params.get("warehouse", str()),
-        )
-        self.engine = sqlalchemy.create_engine(self.connection_url)
+
```

Comment:
I think this causes flake8 in my pre-commit.ci run to complain could you remove that blank line?

Reply:
Done! 🥳